### ISSUE TITLE
GLK Peer Review QA - Issue 10

### ISSUE THREAD

**Issue by gkerber** [Helen1479/FGHJ4567.github.io#13]
10.
Page Title: The Plant Importance
Filename/URL:   https://helen1479.github.io/FGHJ4567.github.io/Plant.html
Type of Issue: Validity – Accessibility
Description of Issue:  Page failed accessibility testing:
**Accessibility Insights for Web** 
2 failures were detected:
html-has-lang: <html> element must have a lang attribute (1) WCAG 3.1.1
list: <ul> and <ol> must only directly contain <li>, <script> or <template> elements (1) WCAG 1.3.1
